app/bundles/FormBundle/Form/Type/FormType.php

```diff
@@ -106,6 +106,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ],
         ]);
 
+        $builder->add('inContactTab', 'yesno_button_group', [
+            'label' => 'mautic.form.form.contacttab',
+            'attr'  => [
+                'tooltip' => 'mautic.form.form.contacttab.tooltip',
+            ],
+            'data'=> $options['data']->isInContactTab() ? true : false,
```

Comment:
One more thing. If the `isInContactTab` method already returns boolean, lets just use that result instead of another true/false values.

Reply:
You're right. Now hope it's ready to test ;)